src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -412,6 +412,17 @@ protected function sendEarlyHints(iterable $links = [], ?Response $response = nu
         return $response;
     }
 
+    /**
+     * Call FormInterface::handleRequest() and return the result of the condition FormInterface::isSubmitted() and FormInterface::isValid().
+     */
+    protected function isSubmittedFormValid(FormInterface $form): bool
```

Comment:
Honestly, I don't like this alternative. It makes it less obvious that we're dealing with a request here. 

Since all methods involved concern the form only, I don't see why the controller should communicate with the form when the form can communicate directly with the request, making this shortcut helpful for controllers not extending `AbstractController`.

Reply:
Like my old PR: https://github.com/symfony/symfony/pull/54726 ?